pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -752,6 +758,31 @@ func (k *Bootstrapper) GenerateToken(cc config.ClusterConfig) (string, error) {
 	joinCmd := r.Stdout.String()
 	joinCmd = strings.Replace(joinCmd, "kubeadm", bsutil.InvokeKubeadm(cc.KubernetesConfig.KubernetesVersion), 1)
 	joinCmd = fmt.Sprintf("%s --ignore-preflight-errors=all", strings.TrimSpace(joinCmd))
+
+	if controlPlane {
+		conf := bsutil.KubeadmYamlPath
+		certKeyCmd := exec.Command("/bin/bash", "-c", fmt.Sprintf("%s init phase upload-certs --config %s --upload-certs", bsutil.InvokeKubeadm(cc.KubernetesConfig.KubernetesVersion), conf))
+		r, err = k.c.RunCmd(certKeyCmd)
+		if err != nil {
+			return "", errors.Wrap(err, "getting cert key")
+		}
+		certKey := r.Stdout.String()
+		var lines []string
```

Comment:
I believe this code could this be simplified to: `lines := strings.Split(r.Stdout.String(), "\n")`

Also, move the declaration of `lines` as close as possible to its first usage.

Reply:
done